aiosenseme/device.py

```diff
@@ -113,7 +114,7 @@ def is_closing(self) -> bool:
             return True  # opened connection but no transport is closed
         return self.transport.is_closing()
 
-    async def receive(self) -> str:
+    async def receive(self) -> str | None:
```

Comment:
Do we need to import `annotations` from `__future__` to support this expression? This library seems to support Python 3.6 still, although that's end of life. Maybe that needs to be changed first?

Reply:
Good question...

Depends if the library owner still wants to support Python 3.6 or not.

I'm happy to adjust this based on the decision to support 3.6 or drop support for it.

Reply:
As far as I can tell there is no need to continue supporting Python 3.6. @frenke if you want to make the changes accordingly...

Seeing how I no longer have access to these fans someone else will need to do the testing for these changes. Maybe @bdraco would like to assume responsibility for this project too.

Reply:
@mikelawrence I can take this if you send me a github admin invite to this repo and PyPI access.

My handle is the same on PyPI

Reply:
Thanks.  I'll get some CI cleanups done in the next 48 hours and then get this merged

Reply:
@bdraco I can transfer ownership of the repository if you would like to handle things that way.

Reply:
That would be great 👍

Reply:
Github complains when I try to transfer ownership...

bdraco/aiosenseme already exists

Might need to rename or remove.

Reply:
Should be good now

Reply:
@mikelawrence apologizes if I missed some approval or button I need to hit if not it should be clear to do the transfer now

Reply:
@bdraco  Github says "This repository is being transferred to @bdraco."

According to [Github docs](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) you should have received an email with instructions to complete the transfer.

I can abort the transfer and try again.

Reply:
Looks like it ended up in SPAM for some reason. Unfortunately it looks like it needs to be resent because `Oops! That repository transfer has expired.`. Sorry for the extra steps

Reply:
@bdraco I restarted the transfer process.

Reply:
@bdraco no worries about the extra steps. Thanks for picking this up this project.

Reply:
Looks like it was successful. Thank you!

Reply:
Cool! Now what? 😬

Reply:
I'm going to something like this:

- repo cleanup
- CI for the repo
- github maint
- remove python 3.6 support
- come back to this,
- cleanups to auto comfort mode
- processing outstanding pr
- release
- address some of the feedback on the Home Assistant PRs.

Reply:
👍